inspirehep/modules/workflows/workflows/article.py

```diff
@@ -148,6 +154,7 @@
                 previously_rejected(),
                 [
                     mark('already-ingested', True),
+                    mark('previously-rejected', True),
```

Comment:
This will only require a migration if there's a workflow stopped in that group of tasks (that is, the mark block), so I think it's safe.
It also helps a lot on debugging why a workflow was discarded.